config/rbac/role.yaml

```diff
@@ -120,3 +120,15 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
Please get rid of this.
make manifests will generate this based on "comment marker" 
https://github.com/keikoproj/addon-manager/pull/102/files

Reply:
This was created by `make manifests` actually the builder comment will generate this and we expect this to be committed so it's was missing and should be added.